pkg/api/v1/conversion.go

```diff
@@ -139,17 +140,42 @@ func addFastPathConversionFuncs(scheme *runtime.Scheme) error {
 	return nil
 }
 
+func Convert_v1_Service_To_api_Service(in *Service, out *api.Service, s conversion.Scope) error {
+	// If the user specifies an alpha annotation, move it to beta (#33625).
```

Comment:
Vs just clobbering? I'm trying to avoid:    
user upgrades 
we just copy alpha to beta and delete alpha
user has working cluster with lbs
we issue PSA and rollback all clusters with daemonsets
users working lb breaks because old cluster doesn't understand beta